lib/include/jxl/encode.h

```diff
@@ -66,11 +66,49 @@ typedef enum {
    */
   JXL_ENC_NEED_MORE_OUTPUT = 2,
 
-  /** The encoder doesn't (yet) support this.
+} JxlEncoderStatus;
+
+/**
+ * Error conditions:
+ * API usage errors have the 0x80 bit set to 1
+ * Other errors have the 0x80 bit set to 0
+ */
+typedef enum {
+  /** No error
    */
-  JXL_ENC_NOT_SUPPORTED = 3,
```

Comment:
Deprecate this value (stop using it) but don't remove it. Reusing value 3 for the next return code could cause problems.

Reply:
Updated.